week-1/exercises/decimal.js

```diff
@@ -1,6 +1,14 @@
 
 const num = 56.5467;
 
+const wholeNumberPart = Math.round(num) - 2;
+const decimalPart = num - Math.round(num);
+const roundedNum = Math.round(num) - 1;
```

Comment:
To round a number down, you use `Math.floor(num)`

See the documentation here: [https://www.w3schools.com/js/js_math.asp](https://www.w3schools.com/js/js_math.asp)